Makefile

```diff
@@ -34,7 +34,6 @@ models: \
 		lvwiki_models \
 		nlwiki_models \
 		nowiki_models \
-		nowiki_models \
```

Comment:
I don't know how it got there but it's redundant and the auto-generation removed it, same for other removals.